### DISCUSSION TITLE
Issue, Pull request Labels

### DISCUSSION THREAD

**Discussion by MMMIIIN** [DeveloperAcademy-POSTECH/MC2-Team5-Firefighter#319] [Ideas]
## 작성 이유 
<img width="925" alt="image" src="https://user-images.githubusercontent.com/78677571/196026078-f3509ed9-f528-4bec-b5b0-60a5a3bf83c0.png">

Async Swift 002에서 29cm iOS리드개발자 김우성님의 세션을 듣고, 그 중 일부였던 PR Label에 대해 적용해보기 위함입니다.
[Async Swift 우성님의 세션 노션 정리 링크](https://deeply-tea-3fa.notion.site/7397ba7addd54192b01fad06b6c4222e)
잘 정리하진 못했지만 너무 좋은 세션이었어서 공유드립니다 ^_______^

## Issue와 Pull request에서 사용하는 Label들을 정리합니다.

| Label |  Description  |
| :--- | :--- | 
| <img width="408" alt="스크린샷 2022-10-10 오후 9 13 07" src="https://user-images.githubusercontent.com/78677571/194863841-3417939c-64d3-481a-b7d3-32d8122d495a.png"> | 버그를 수정했을 때 사용  |  
|<img width="408" alt="스크린샷 2022-10-10 오후 9 16 43" src="https://user-images.githubusercontent.com/78677571/194864413-5e580d05-b99a-4286-aa76-41eb42ab41ae.png"> | 새로운 기능을 추가했을 때 사용 |
| <img width="408" alt="스크린샷 2022-10-10 오후 9 17 34" src="https://user-images.githubusercontent.com/78677571/194864589-5a1f0453-e690-4401-93ec-674d94dbb0f5.png"> | 개발 중 팀원들의 도움이 필요할 때 사용 |
| <img width="408" alt="스크린샷 2022-10-10 오후 9 18 25" src="https://user-images.githubusercontent.com/78677571/194864713-ad6624a7-b5d0-4f7a-a391-0bc3cdf0eee3.png"> | 무조건적인 칭찬을 받고싶을 때 사용 <br> (팀원들의 사기진작을 위해 무조건 칭찬하는 라벨) |
| <img width="408" alt="스크린샷 2022-10-10 오후 9 36 36" src="https://user-images.githubusercontent.com/78677571/194867681-cebb8fe2-6850-41ea-914e-1b7a711f9694.png"> | 릴리즈 브랜치에서 정식배포가 가능한 상태의 코드를 메인 브랜치로 머지할 때 사용 |
| <img width="408" alt="스크린샷 2022-10-10 오후 9 21 53" src="https://user-images.githubusercontent.com/78677571/194865272-377e794f-6d10-4c94-a4eb-23e276002bb3.png"> | 당장 급한 PR은 아닌 경우에 사용 |
| <img width="408" alt="스크린샷 2022-10-10 오후 9 22 33" src="https://user-images.githubusercontent.com/78677571/194865381-59a49892-7e87-4e2b-ae57-0a298bcc23bb.png"> | API 통신 관련 작업을 했을 때 사용 |
| <img width="408" alt="스크린샷 2022-10-10 오후 9 23 21" src="https://user-images.githubusercontent.com/78677571/194865493-1ecf8ee8-bdab-47f9-855b-22544ab7ba27.png"> | 당장 급하니까 최대한 빨리 리뷰를 부탁할 때 사용 <br> **이 라벨이 붙었을 땐 가능한 한 빨리 리뷰하기 !** | 
| <img width="408" alt="스크린샷 2022-10-10 오후 9 34 39" src="https://user-images.githubusercontent.com/78677571/194867375-a84ff208-7bfb-4bb9-b1b8-66c2bd783248.png"> | 큰 작업은 아니지만 사소한 무언가를 수정했을 때 사용 |
| <img width="408" alt="스크린샷 2022-10-10 오후 9 38 57" src="https://user-images.githubusercontent.com/78677571/194868076-deb44480-d501-4978-988e-3d3b2bd3227b.png"> | 어떤 뷰를 작업했는지 표시하기 위해 사용 |
| <img width="408" alt="스크린샷 2022-10-10 오후 9 40 40" src="https://user-images.githubusercontent.com/78677571/194868487-a5d0db76-8eae-443b-ad5a-7576d231d728.png"> | 초기에 프로젝트 세팅할 때 사용 |
| <img width="408" alt="스크린샷 2022-10-10 오후 9 41 11" src="https://user-images.githubusercontent.com/78677571/194868586-2ae1f966-982f-441f-b606-7a3686184e08.png"> | 자동화 관련 무언가를 했을 때 사용 |
| <img width="309" alt="image" src="https://user-images.githubusercontent.com/78677571/195507906-e12211ba-8007-4595-9382-f849ef75fcce.png"> | 새로운 실험을 할 때 사용 |

추가해야할 라벨들이 있다면 생성 후 위와 같이 추가해주세요 ! ^________^

**Comment by YoonAh-dev** [comment]
확인했습니다. 정리해주셔서 감사합니다.👍
전에 사용했던 라벨들보다 좋은 거 같네요. 많이 배우고 갑니다.

**Reply by MMMIIIN** [reply]
감사합니다 ^______^

**Reply by YoonAh-dev** [reply]
칭챤합니다.

**Comment by dangsal** [comment]
확인했습니다. 

아주 쓸법한 라벨들이라 굉장히 기대되네요 감사합니다!

**Reply by MMMIIIN** [reply]
감사합니둥 ^________^

**Comment by dangsal** [comment]
진짜 대박사건 이건 짱이야.... 케미 최고입니다!! 대단하십니다 최고!!

**Reply by MMMIIIN** [reply]
감사합니다아~~~ ^_____________^
